src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/Claim.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.civil.model.citizenui;
+
+public interface Claim {
+
+    boolean hasResponsePending();
+
+    boolean hasResponsePendingOverdue();
+
+    boolean hasResponseDueToday();
+
+    boolean hasResponseFullAdmit();
+
+    boolean defendantRespondedWithFullAdmitAndPayImmediately();
+
+    boolean defendantRespondedWithFullAdmitAndPayBySetDate();
+
+    boolean defendantRespondedWithFullAdmitAndPayByInstallments();
+
+    boolean responseDeadlineHasBeenExtended();
+
+    boolean isEligibleForCCJ();
+
+    boolean claimantConfirmedDefendantPaid();
+
+    boolean isSettled();
+
+    boolean isSentToCourt();
+
+    boolean claimantRequestedCountyCourtJudgement();
+
+}
```

Comment:
if we created this interface to use like "constant" I don't have correct word to describe we could use Static Interface Methods
Some thing like that:
public interface Vehicle {
    
    // regular / default interface methods
    
    static int getHorsePower(int rpm, int torque) {
        return (rpm * torque) / 5252;
    }
}

Reply:
but in our case we dont need to use static methods. This is an interface for a claim retrieved from the database for defendant and then potentially for a claimant. It helps us to generalise the ccd and cmc claims so that we could call the methods without distinguishing which one is which

Reply:
I agree